profiles/base.yaml

```diff
@@ -1432,3 +1432,20 @@ duskruin:
 
 #turn on heal/remedy text subs by ingredient name: yelith (limb-IW) tonic
 herb_textsubs: false
+
+# Settings used for wand-watcher, which uses 'wands' (ex: bloodwood branch) based on timers
+# See: https://elanthipedia.play.net/Lich_script_repository#wand-watcher for details
+wands:
+  # magic wand:                     # Adj + Noun of the wand.  Must be unique  [Mandatory]
+  #   activation message: Foo Bar   # Spell activation message                 [Mandatory]
+  #   activation verb: tap          # Verb used to activate the wand           [Optional - defaults to 'tap']
+  #   container: backpack           # Container where the wand(s) are stored   [Mandatory]
+  #   cooldown: 32                  # minutes between activation attempts      [Optional - defaults to 30]
+  #   count: 2                      # number of wands you have                 [Optional - defaults to 2]
+
+wand_watcher_no_use_scripts:  # Don't try to grab any wands while these scripts are running.
+  - burgle
+  - go2
+
```

Comment:
I don't think partial room name matches would be a good idea here, it should require full room titles to prevent over-matching.  

The number of no use rooms that need to be specified that way (rooms which you can't use it, _but can't be mapped_) should be minimal.

Reply:
I hear you. Since this is the user's yaml, they will have the flexibility to put as much of the room's title here as they'd like. So to your point, the `base.yml` and examples should specify a full room's title as maybe the "best practice" and I like for the script to still support the flexibility of `=~` for folks (like me) that may want a broader net cast rather than listing each room's title or id individually.